code/src/App.js

```diff
@@ -1,9 +1,23 @@
-import React from 'react'
+import React from "react";
+import { BrowserRouter, Switch, Route, Redirect } from "react-router-dom";
+import List from "./components/List";
```

Comment:
During my code review with Maksymilian, he mentioned there should be a line break (so enter) between imports from the json and from component local files....